frontend/src/error/ErrorHandler.tsx

```diff
@@ -1,13 +1,13 @@
 import React from "react";
 import { useLocation } from "react-router-dom";
 import { get } from "lodash";
-import { ErrorPage } from "./ErrorPage";
+import { BackendErrorPage } from "./ErrorPages";
 
 const ErrorHandler = ({ children }: any) => {
```

Comment:
We should probably type the ErrorHandler as `React.FC` so that we don't have to explicitly type the props or the return type of the component